src/initial-html.js

```diff
@@ -4,236 +4,229 @@
  */
 
 export default `
-<!-- wp:image -->
-<figure class="wp-block-image"><img alt=""/></figure>
-<!-- /wp:image -->
-
-<!-- wp:media-text -->
-<div class="wp-block-media-text is-stacked-on-mobile"><figure class="wp-block-media-text__media"></figure><div class="wp-block-media-text__content"><!-- wp:paragraph {"className":"has-large-font-size"} -->
-<p class="has-large-font-size"></p>
-<!-- /wp:paragraph --></div></div>
-<!-- /wp:media-text -->
-
-<!-- wp:image -->
-<figure class="wp-block-image"><img src="https://cldup.com/cXyG__fTLN.jpg" alt=""/></figure>
-<!-- /wp:image -->
-
-<!-- wp:title -->
-Hello World
-<!-- /wp:title -->
+<!-- wp:heading -->
+<h2>Text Blocks</h2>
+<!-- /wp:heading -->
 
 <!-- wp:heading -->
 <h2>What is Gutenberg?</h2>
 <!-- /wp:heading -->
 
 <!-- wp:paragraph -->
-<p><strong>Bold</strong> <em>Italic</em> <s>Striked</s> Superscript<sup>(1)</sup> Subscript<sub>(2)</sub></p>
+<p><strong>Bold</strong> <em>Italic</em> <s>Striked</s> Superscript<sup>(1)</sup> Subscript<sub>(2)</sub> <a href="http://www.wordpress.com" target="_blank" rel="noreferrer noopener">Link</a></p>
```

Comment:
Not sure if it can be a problem, but maybe we could change the `wordpress.com` references to `wordpress.org`? To be in the safe side.

Same in the contact info block.